content/docs/user-guide/remote-content/update-a-remote-repository.md

```diff
@@ -0,0 +1,120 @@
+    --repo-name <new repo name>
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+### Add a new Gem or new Gem version to a remote repository, or update an existing Gem
+
+You can use the `o3de` CLI tool to add information from a `gem.json` file to a `repo.json` file and make an archive that you can upload with the following command:
+
+{{< tabs name="Add information about a gem to `repo.json`" >}}
+{{% tab name="Windows" %}}
+
+```cmd
+<engine>\scripts\o3de.bat edit-repo-properties --repo-path <path to repo.json> --add-gems <local gem paths> --release-archive-path <path where the gem archive files will go> --download-prefix <URL prefix for where the gem archive will made available, e.g. https://github.com/o3de/o3de-extras/releases/2.0>
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```cmd
+<engine>/scripts/o3de.sh edit-repo-properties \
+    --repo-path <path to repo.json> \
+    --add-gems <local gem paths> \
+    --release-archive-path <path where the gem archive files will go> \
+    --download-prefix <URL prefix for where the gem archive will made available, e.g. https://github.com/o3de/o3de-extras/releases/2.0>
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+After running this command, the `repo.json` file will be modified so the information inside each `gem.json` file is included and the `sha256` archive file hash is correct.
+
+If a gem with the same name already exists in the `repo.json` but the version is different, the `gem.json` data from the new version will be added to the [`versions_data`](/docs/user-guide/programming/gems/manifest/#gemjson-manifest-contents) field.  If a gem with the same name and version already exists the gem will be updated.  
+
+{{< tip >}}
+After modifying remote repository contents, we recommend you [test the changes locally](#testing-o3de-remote-repository-changes) before uploading them to a public web server or GitHub.
+{{< /tip >}}
+
+
+## Testing O3DE remote repository changes
+
+We recommend that you use the Project Manager and O3DE CLI to test your remote repository before you upload the changes to GitHub, a web server or other source control.
+
+### Test with Project Manager
+
+You can add local remote repositories in the Project Manager and verify the contents appear as expected and downloads succeed.
+1. Add your local remote repository in Project Manager on the **Remote Sources** page on the **Engine** tab, by pressing the **Add Repository** button and then use the browse button to select the folder containing your `repo.json` file and then press the **Add** button.
+1. Select the remote repository in the list and verify the expected contents are listed in the right pane.
+1. Verify any added gems appear correctly in the **Gem Catalog**.
+1. Verify any added projects appear correctly on the **Projects** page.
+1. Verify any added templates appear correctly on the **Create a New Project** page.
+
+### Test with the O3DE CLI 
+
+You can add local remote repositories with the O3DE CLI and verify downloads succeed.
```

Comment:
isn't `local` and `remote` contradictory?

Reply:
It is! But the "remote repository" is actually on your "local" computer, I'll rephrase it so it doesn't sound so wierd.